cpp/src/aztec3/circuits/rollup/root/.test.cpp

```diff
@@ -237,7 +239,7 @@ NT::fr iterate_through_tree_via_sibling_path(NT::fr leaf, NT::uint32 leafIndex,
     return leaf;
 }
 
-template <size_t N> std::array<fr, N> get_sibling_path(stdlib::merkle_tree::MemoryTree tree, size_t leafIndex)
+template <size_t N> std::array<fr, N> get_sibling_path(MemoryTree tree, size_t leafIndex)
```

Comment:
should we move this to base/utils.cpp? Since we have the same method in base rollup

Reply:
Should essentially be a function on the tree i think, similar to it having a hashpath, will let it be for now.